### DISCUSSION TITLE
Log cooking - other functions such as comments and list of entries available?

### DISCUSSION THREAD

**Discussion by sg17843** [TandoorRecipes/recipes#1468] [Q&A]
Hi all,

I like the "Log cooking" function. However, in the meantime I have noticed that I am missing some functions ... are there already ideas for improvements? 

For example, I would like to have a **comment field** for additions such as "with friends / family", "with starter / dessert" or others.
Furthermore, it would be useful to have the possibility to report "when what was cooked". 

KR

**Comment by smilerz** [comment]
A feature request to add a text field on the log is here #706 

What do you mean by "when what was cooked"

**Reply by sg17843** [reply]
In the simplest case, a chronological list of recipes. 
In this list (formats e.g. html, text or PDF) a search function could then be used to find, for example, in the comments, which meals I last cooked with my friends/family.

**Reply by smilerz** [reply]
The development to search/sort for a recipe based on when you last cooked it is complete.  It will be in the next release.

**Reply by sg17843** [reply]
Understood that there are further developments ... I'll wait for the upcoming releases. Thank you.

**Reply by smilerz** [reply]
Wasn't looking to shut you down - just letting you know that particular feature is close :-)
